gateway-server/src/main/java/org/apache/knox/gateway/websockets/ProxyWebSocketAdapter.java

```diff
@@ -87,6 +87,10 @@ public void onWebSocketConnect(final Session frontEndSession) {
      * plumbing takes place
      */
     container = ContainerProvider.getWebSocketContainer();
+    container.setDefaultMaxTextMessageBufferSize(frontEndSession.getPolicy().getMaxTextMessageBufferSize());
```

Comment:
Sorry about the delay, meant to submit it yesterday but the review was in "pending" state, apparently I don't git well :(

Reply:
@moresandeep Yes, the values that is being set for the servlet factory get applied to any session for the frontEndSession. I'm just copying them to client container. Since the container is the generic type I can't actually set the max text message size policy but I've looked at the Jetty code and setting the buffer also sets the message size.